src/main/java/com/google/jenkins/plugins/computeengine/client/ComputeClient.java

```diff
@@ -77,6 +67,10 @@ public void setCompute(Compute compute) {
         this.compute = compute;
     }
 
+    public Compute getCompute() {
```

Comment:
We should avoid breaking encapsulation like this, it introduces inconsistency to the code base.